Sources/NIOHTTP1/HTTPDecoder.swift

```diff
@@ -12,7 +12,12 @@
 //
 //===----------------------------------------------------------------------===//
 
+#if compiler(>=6.1)
```

Comment:
Why is this `>=6.1`? We can do imports with access level from 6.0 IIRC.

Reply:
For no better reason than 6.1 is where the errors started to be observed.